css/style.css

```diff
@@ -0,0 +1,146 @@
+/* GENERAL STYLING */
+
+.crimson-font-color{
+  color: #540002;
+}
+
+.link, .link:visited{
+  color: #666F75;
+  opacity: 0.6;
+}
+
+.link:hover, .link:focus, .link:active {
+  text-decoration: none;
+  opacity: 1;
+  color: #636C72;
+}
+
+.item {
+  padding-bottom: 40px;
+}
+
+.page-head {
+  padding: 50px 0;
+}
+
+/* STYLE CONTAINERS */
+
+.cover .container{
+  padding-top: 200px; 
+}
+
+body, html {
+  height: 100%;
+}
+
+#about{
+  margin-bottom: 150px;
+  height: auto;
+  background: white;
+  color: #540002;
+}
+
+/* BUTTON PROPERTIES */
+
+.btn-info{
+  background: #74FF11;
+  color: #540002 !important;
```

Comment:
Never ever ever ever ever use `!important`. It's messy & difficult to maintain. Use CSS's specificity rules to specify what rules should override what rules. If you're trying to override bootstrap, just make sure your CSS selector is more specific than the one bootstrap uses (can be found via Chrome Dev Tools).